models/repo.go

```diff
@@ -1450,7 +1450,14 @@ func GetRepositoryByID(id int64) (*Repository, error) {
 
 // GetUserRepositories returns a list of repositories of given user.
 func GetUserRepositories(userID int64, private bool, page, pageSize int) ([]*Repository, error) {
-	sess := x.Where("owner_id = ?", userID).Desc("updated_unix")
+	sess := x.Where("owner_id = ?", userID).Desc("uid").Desc("updated_unix")
```

Comment:
For a better readability and to get the real line / instruction in case of error, please set  XORM instruction per line

eg :

```
x.Where("owner_id !=? ", user.ID).
        Desc("uid").
        Desc("updated_unix")
```
or

```
x.
        Where("owner_id !=? ", user.ID).
        Desc("uid").
        Desc("updated_unix")
```